### ISSUE TITLE
Add toggle control for tokens to ignore grid.

### ISSUE THREAD

**Issue by Taliesin1** [encounterplus/encounterplus#466]
This does not need to be in the pop-up contextual menu, but it would be nice to have in the token settings panel.

**Comment by Splinter714** [comment]
Do you mean for them to ignore the grid on a per-token basis? I’m assuming you’re not talking about the magnet tool on the map toolbar.

**Comment by Taliesin1** [comment]
Correct. I'd like certain (most) tokens to abide by the Magent Tool setting, but I'd like to occasionally be able to choose certain tokens to ignore the grid, even though all the other token are bound to it.
